cmd/nokernelinfo.go

```diff
@@ -0,0 +1,9 @@
+// +build !linux
```

Comment:
Please rename this file to `kernelinfo_notlinux.go` and keep the build tag.

Reply:
Man, go's build tags are getting messy :D - some can be expressed by file names, but only a subset...